lib/absinthe/language/input_value_definition.ex

```diff
@@ -29,12 +29,15 @@ defmodule Absinthe.Language.InputValueDefinition do
         description: node.description,
         type: Blueprint.Draft.convert(node.type, doc),
         identifier: Macro.underscore(node.name) |> String.to_atom(),
-        default_value: Blueprint.Draft.convert(node.default_value, doc),
+        default_value: default_value(node),
```

Comment:
Without this function, the `default_value` looks like this:

```
%Absinthe.Blueprint.Input.Integer{errors: [], flags: %{}, schema_node: nil, source_location: %Absinthe.Blueprint.SourceLocation{column: 18, line: 28}, value: 10}
```

The function here probably needs to be smarter than what makes an integer work

cc/ @bruce

Reply:
This might be able to piggy-back off some of the work I'm doing to make directive arguments in schema definitions work (in #741). Put a pin in it for now and we can build another PR later.